src/workers/atlas_worker.js

```diff
@@ -144,6 +151,18 @@ export default class AtlasWorker extends PeriodicWorker {
     return true;
   }
 
+  async isEnoughAvailableDiskSpace() {
```

Comment:
There's a very visible duplication between `isEnoughAvailableDiskSpace` and `isEnoughFundsToPayForGas`. I think it's worth abstracting.